tests/integration/workflows/helpers/calls.py

```diff
@@ -167,6 +167,28 @@ def generate_record():
     return json_data
 
 
+def insert_citing_record():
+    """Provide record fixture."""
+    json_data = json.loads(pkg_resources.resource_string(
+        __name__,
+        os.path.join(
+            '../fixtures',
+            'oai_arxiv_citing_record.json'
+        )
+    ))
+
+    categories = {'core': [], 'non-core': []}
+    for eprint in json_data.get('arxiv_eprints', []):
+        categories['core'].extend(eprint.get('categories', []))
+
+    if 'preprint_date' in json_data:
+        json_data['preprint_date'] = datetime.date.today().isoformat()
```

Comment:
MINOR it would have been cleaner to use https://github.com/spulec/freezegun here instead, but it's OK for now.

Reply:
Just copied from: https://github.com/inspirehep/inspire-next/blob/ac4fdbbf50b03b55dd5f4983600ec65cca4879d8/tests/integration/workflows/helpers/calls.py#L170
Again, not sure why we need to do this.